### ISSUE TITLE
dynamically added SVG mask has wrong(HTMLUnknownElement) DOM type 

### ISSUE THREAD

**Issue by IL55** [facebook/react#4535]
Hi, I found strange issue connected with svg mask elements, i.e.
```      
      <defs>
           <mask id="myMask"> .... </mask>
      </defs>
```
when mask adds dynamically (i.e. after, for example, button click)
mask has DOM "type" HTMLElement/HTMLUnknownElement, I can see in chrome DevTools for svg.defs.mask[1] properties
```
mask#Mask1
HTMLUnknownElement
HTMLElement
Element
Node
EventTarget
Object
```
But for mask, created "statically" i.e. on load I see DOM type as SVGElement/SVGMaskElement (svg.defs.mask[0]):
```
mask#Mask0
SVGMaskElement
SVGElement
Element
Node
EventTarget
Object
```

Here is jsfiddle to repeat the problem:
https://jsfiddle.net/zzbm3qar/

here is same bug description but for jquery
http://stackoverflow.com/questions/29401475/dynamically-add-a-shape-mask-to-an-inline-svg

**Comment by jimfb** [comment]
Possibly a namespacing issue?  I have no reason to say that, other than it sounds intuitively reasonable to me.  (May therefore be tangentially related to https://github.com/facebook/react/issues/3420)

**Comment by IL55** [comment]
Yes thanks, it seems that wrong namespace bug, I found fix for last stable (0.13.3) but I can't apply it to master (and create PR) due to `src/shared/vendor/core/getMarkupWrap.js` removed (rewrote) completely? It seems in this pull request https://github.com/facebook/react/pull/3808

But someone can continue use and support original stable 0.13.x, and that fix could be helpful for him.
So fix for last stable is next:
add in file `getMarkupWrap.js`
to `shouldWrap` object next property: `  'mask': true ` and to `markupWrap` next property ` 'mask': svgWrap `

**Comment by zpao** [comment]
`getMarkupWrap` was moved out of this repo and is now a dependency. It already supports `mask` (https://github.com/facebook/fbjs/blob/master/src/core/getMarkupWrap.js#L71). I don't think we're going to backport to 0.13 (I don't foresee us doing any more releases there).
